src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/wwwroot/theme-2019/css/responsive.css

```diff
@@ -1,3 +1,17 @@
+@media screen and (min-width: 2000px) {
+    .countdown_section {
+        padding-bottom: 323px;
+        padding-top: 320px;
+    }
+}
+
+@media screen and (min-width: 1200px) and (max-width: 1999px) {
+    .countdown_section {
+        padding-bottom: 150px;
+        padding-top: 150px;
+    }
+}
+
```

Comment:
this chunk of code changes padding to make Santa visible on HD, fullHD, and 4K monitors - I think it is done for now (although 4K could use more changes to look good)